avalon/tools/sceneinventory/model.py

```diff
@@ -228,7 +228,11 @@ def add_items(self, items, parent=None):
             # Get the primary family
             family = version["data"].get("family", "")
```

Comment:
@davidlatwe how about we push this into the schema if/else too. Because in this case the `version.data["family"]` will still overwrite the `subset["data"]["families"]` for the primary family although it should actually be the other way around.

Reply:
Ah, so if the schema is `subset-3.0`, then just take from `subset["data"]["families"]`, and consider `version["data"]["family"]` if not.
Yeah, this is better !

Reply:
Okay, I have this fixed in commit 0c548f8, and the readability should be suffice I think :)